engines/bastion_katello/app/assets/javascripts/bastion_katello/capsule-content/capsule-content.routes.js

```diff
@@ -26,4 +26,12 @@ angular.module('Bastion.capsule-content').config(['$stateProvider', '$urlRouterP
         }
     });
 
+    $urlRouterProvider.otherwise(function ($injector) {
```

Comment:
Just one note, this line globally overrides bastion's default otherwise method:
https://github.com/Katello/bastion/blob/master/app/assets/javascripts/bastion/bastion.module.js#L60
Is that intentional?